students/ian_letourneau/Lesson06/test_mailroom.py

```diff
@@ -0,0 +1,53 @@
+from mailroom import send_thank_you, create_report, send_all
+import pytest
+
+# Test the send_thank_you function: basic
+
+
+def test_1():
+    assert send_thank_you("Angelina Jolie", 3.47) == ('Dear Angelina Jolie,'
+                                                      ' we wanted to say thank'
+                                                      ' you for your generous'
+                                                      ' donation of $3.47!')
+
+# Test the send_thank_you function: long name
+
+
+def test_2():
```

Comment:
you should avoid naming your functions with vague names like this, when you get a good amount of tests debugging becomes difficult since names mean nothing.

your test names should have some indication what it is testing, for example
```
def test_send_thank_you_two_param(): pass
def test_send_thank_you_three_param(): pass
```